src/templates/en/base.html

```diff
@@ -24,6 +24,11 @@
   You’re viewing a previous edition of the Web Almanac.
   Check out the <a href="/{{ lang }}/{{ DEFAULT_YEAR }}/{% if chapter_lang_exists(lang, DEFAULT_YEAR, slug) or page_lang_exists(lang, DEFAULT_YEAR, slug) %}{{ slug }}{% endif %}">{{ DEFAULT_YEAR }} edition!</a>
 </p>
+{%- elif year == DEFAULT_YEAR -%}
+<p lang="en">
```

Comment:
Let's remove these `lang` attributes (and from line 23) when they match the template language:

```suggestion
<p>
```

This was my fault as I never removed them when translating the text. But let's do it properly now.